### ISSUE TITLE
react-static cli create error

### ISSUE THREAD

**Issue by frankolson** [nozzle/react-static#172]
The `react-static` cli is producing the following error when running the `create` command:

```
$ react-static create react-static-test-blog

? What should we name this project? react-static-test-blog
? Select a template below... styled-components
=> Creating new react-static project...
(node:5774) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: _chalk2.default.hex is not a function
(node:5774) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

This seems to be happening for every template. I also used yarn: `yarn global add react-static` to install the cli.

**Comment by EmilTholin** [comment]
Hi @frankolson!

Interesting. Could you try to re-install it, but with `npm i -g react-static@4.2.1`, and see if you get the same error?

**Comment by frankolson** [comment]
`npm i -g react-static@4.2.1` worked but `yarn global add react-static@4.2.1` did not

**Comment by gagan0723** [comment]
Having issue with ```npm i -g react-static@4.2.1```. It says
> npm ERR! enoent This is related to npm not being able to find a file.

Any other package get installed safely.

**Comment by tannerlinsley** [comment]
Fixed in v4.2.3
